websites/R/Replit/dist/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "Ashutosh",
+    "id": "316943845596200960"
+  },
+  "service": "Replit",
+  "altnames": ["Replit", "Repl.it", "Repl"],
+  "description": {
```

```suggestion
  "altnames": ["Repl.it"],
  "description": {
```

First one is just the service name....
Third one is the first characters of the service name so this wont even work since its nothing different.